cli/proc_state.rs

```diff
@@ -495,7 +495,10 @@ impl ProcState {
     referrer: &str,
     permissions: &mut PermissionsContainer,
   ) -> Result<ModuleSpecifier, AnyError> {
-    if let Ok(referrer) = deno_core::resolve_url_or_path_deprecated(referrer) {
+    // TODO(bartlomieju): ideally we shouldn't need to call `current_dir()` on each
+    // call - maybe it should be caller's responsibility to pass it as an arg?
```

Comment:
I'm not sure about either of these - that's why I left a TODO here to proceed with the refactor of `deno_core` - nothing really changes here besides acquiring CWD lifted here instead of being done implicitly by `deno_core`.